internal/process/defunct_processes.go

```diff
@@ -50,11 +50,11 @@ func DefunctProcessesForPath(path string) (defunctCount uint, retErr error) {
 
 		stat, err := processStats(path, name)
 		if err != nil {
-			logrus.Debugf("Failed to get the status of process with PID %s: %v", name, err)
+			logrus.Warnf("Failed to get the status of process with PID %s: %v", name, err)
```

Comment:
**To to reviewer:**

Even though I moved this log line to the warning level, I am still unsure about how useful this line would also be here. In the end, this is just the metrics collector code.